eng/common/scripts/Verify-Links.ps1

```diff
@@ -6,18 +6,20 @@ param (
   # switch that will enable devops specific logging for warnings
   [switch] $devOpsLogging = $false,
   # check the links recurisvely based on recursivePattern
-  [switch] $recursive = $true,
+  [switch] $recursive,
   # recusiving check links for all links verified that begin with this baseUrl, defaults to the folder the url is contained in
   [string] $baseUrl = "",
   # path to the root of the site for resolving rooted relative links, defaults to host root for http and file directory for local files
   [string] $rootUrl = "",
   # list of http status codes count as broken links. Defaults to 400, 401, 404, SocketError.HostNotFound = 11001, SocketError.NoData = 11004
   [array] $errorStatusCodes = @(400, 401, 404, 11001, 11004),
-  # flag to allow resolving relative paths or not
-  [bool] $resolveRelativeLinks = $true
+  # flag to allow checking against azure sdk link guidance.
+  [bool] $linkGuidance = $true
 )
 
 $ProgressPreference = "SilentlyContinue"; # Disable invoke-webrequest progress dialog
+# list of locale keywords
+[array] $locales = @("/en-us/")
```

Comment:
I wouldn't make this a list I would just make it a regex.

Reply:
We might add more locale strings in future. Make it regex gives us limited room to extend, but I am flexible on the data structure here.

Reply:
Adding more locales can be added to the regex but if that doesn't end up working for a future scenario then we can make adjustments then. For now I'd like to keep it as simple as possible.